src/components/SQFormDialogStepper/SQFormDialogStepper.js

```diff
@@ -204,16 +184,17 @@ export function SQFormDialogStepper({
             <Divider />
             {steps.length > 1 && (
               <Grid container classes={stepperClasses}>
-                <Stepper nonLinear activeStep={activeStep} classes={classes}>
+                <Stepper activeStep={activeStep} classes={classes}>
                   {steps.map((child, index) => (
-                    <Step key={child.props.label}>
-                      <StepButton
-                        onClick={handleStep(index)}
-                        completed={completed[index]}
-                      >
+                    <Step
+                      key={`${child.props.label}-${index}`}
+                      classes={classes.label}
+                      alternativeLabel
+                    >
+                      <StepButton onClick={handleStep(index)}>
                         <Typography
                           variant="overline"
-                          color={index === activeStep ? 'error' : ''} // sets the color to orange if current step
+                          color={index === activeStep ? 'error' : 'initial'} // sets the color to orange if current step
```

Comment:
This doesn't change the color at all but resolves a console warning because the color being set to '' was not allowed.